main.css

```diff
@@ -76,16 +76,14 @@ body {
 }
 
 .active {
-  background-color: #43a0488c;
+  background-color: #0000ee;
   color: #fff;
 }
 
 .disabled {
-  color: -internal-light-dark(rgba(16, 16, 16, 0.3), rgba(255, 255, 255, 0.3));
-  background-color: -internal-light-dark(
-    rgba(239, 239, 239, 0.3),
-    rgba(19, 1, 1, 0.3)
-  );
+  color: #fff;
+  background-color: #0000ee;
```

Comment:
'active' and 'disabled' classes have the same background color?

Also, are these keywords describing the same behaviour for the keywords outlined in [CSS Pseudo selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes)?